### DISCUSSION TITLE
Suppress Build Warnings

### DISCUSSION THREAD

**Discussion by memjr** [microsoft/azuredatastudio#15148] [Q&A]
Hello, quick question.  We started using Azure Data Studio to work with database projects and when building we get build warnings, more specifically warnings that we don't care about at this time, such as SQL71558 "reference differs only by case from object difinition".

I've edited our .sqlproj file manually and added the bit in bold shown below.  While this works, I wanted to ask if this is good practice or if there is a more appropriate way to handle this (other than changing the code, which we eventually will).

`<?xml version="1.0" encoding="utf-8"?>`
`<Project.......>`
`.....`
`  <PropertyGroup>`
`    <NoWarn>$(NoWarn);SQL71558</NoWarn>`
`  </PropertyGroup>`
`.....`
`</Project>`

**Comment by cubriq** [comment]
Hi Mario, I have exactly the same problem, but your solution does not work for me. Still get the 71558 warnings. Which version of Azure Data Studio are you using? Anything else I need to configure to supress the warnings? Best Regards / Richard
